File: initialApp-master/RentApp/Models/Entities/Vehicle.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace RentApp.Models.Entities
{
    [Table("Vehicles", Schema = "dbo")]
    public class Vehicle
    {
        public int Id { get; set; }
        public int ServiceId { get; set; }
        public string TypeVehicle { get; set; }
        public string Producer { get; set; }
        public string Model { get; set; }

        [Column("ProductionDate", TypeName = "datetime2")]
        public DateTime ProductionDate { get; set; }
        public List<string> Pictures { get; set; }
        public double Price { get; set; }
        public string Description { get; set; }
        public bool Available { get; set; }

    }
}

File: RentClient/RentAppClient/src/app/models/Vehicle.ts
export class Vehicle {
    Id: number;
    ServiceId: number;
    TypeVehicle: string; 
    Producer: string; 
    Model: string;
    Pictures: string;
    Picture: string;
    Description: string;
    ProductionDate: Date;
    Pirce: number;
    
    constructor(serviceId: number, typeVehicle: string, producer:string, model: string, pictures: string, picture: string, 
        description: string, productionDate: Date, pirce: number){
        this.ServiceId = serviceId;
        this.TypeVehicle = typeVehicle;
        this.Producer = producer;
        this.Model = model;
        this.Pictures = pictures;
        this.Picture = picture;
        this.Description = description;
        this.ProductionDate = productionDate;
        this.Pirce = pirce;
    }
}